.travis.yml

```diff
@@ -0,0 +1,22 @@
+dist: bionic
+env:
+  global:
+    - LOCAL_IMAGE="graph-notebook:local"
```

Comment:
I'd rename it to `graph-notebook-docker:local`.

Reply:
Yes I agree it makes the naming more consistent.